src/sign/index.ts

```diff
@@ -41,12 +40,18 @@ export function subscribe(email: string, address: string) {
   );
 }
 
-export function verifySubscribe(email: string, address: string, signature: string) {
+export function verifySubscribe(
+  email: string,
+  address: string,
+  signature: string,
+  signer?: string
+) {
   return verify(
     {
       email,
       address: getAddress(address)
     },
+    signer || wallet.address,
```

Comment:
it should be only the signer.
```suggestion
    signer,
```
you already [check](https://github.com/snapshot-labs/envelop/pull/52/files#diff-6b984d0f3bee25da0d949d2e9fddd45f75509458a9b0c05b589af909ffe0911aR31) its existence so it will be 100% present

Reply:
It's also used here https://github.com/snapshot-labs/envelop/pull/52/files#diff-6b984d0f3bee25da0d949d2e9fddd45f75509458a9b0c05b589af909ffe0911aR43 , exactly same data, but signed by the backend.

This may be confusing, should have a dedicated function for this I think